### PR TITLE
Feature/update string name day

### PR DESCRIPTION
**Pull request by SirJetboy**

### CODE REVIEW

File: lib/los_santos/extensions/string.rb [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # implement method name_day? to the class String
+require 'rubygems'
```
Comment (albertlb) [inline lib/los_santos/extensions/string.rb]: ?

File: lib/los_santos/extensions/string.rb [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+require 'active_support/inflector'
+
 class String
   def name_day?(date = Time.now.localtime)
-    LosSantos.name_day(date).include? self
+    LosSantos.name_day(date).each do |name|
+      return true if name.parameterize == self.parameterize
```
Comment (albertlb) [inline lib/los_santos/extensions/string.rb:7]: why is better parameterize than I18n?

File: lib/los_santos/extensions/string.rb (hunk 1 above)
Comment (SirJetboy) [reply lib/los_santos/extensions/string.rb]: I forgot to remove it

File: lib/los_santos/extensions/string.rb (hunk 2 above)
Comment (SirJetboy) [reply lib/los_santos/extensions/string.rb:7]: I didn't find the perfect function in i18n, there is _transliterate_ but if there is a space or a hyphen, it is gonna stay a space or an hyphen. For example, if I want to return *true* in the comparison between "JEAN BAPTISTE" and "Jean-Baptiste", with _transliterate_ it is not possible, but with parameterize.

Comment (albertlb) [reply lib/los_santos/extensions/string.rb:7]: but parameterize will change "caño" for "cano". And for the "-" you were spliting by "-", which was good. 

Eg. There is no name day for "Anne-Laure" but we have indeed day name for "anne" and "laure"

**Review by albertlb — APPROVED** [review]

**Pull request merged**